core/src/main/java/fj/data/Set.java

```diff
@@ -516,6 +516,30 @@ public final int size() {
   }
 
   /**
+   * Searches the set for an element equal (by the set's Ord) to the one provided. Similar to {@link #split(A)},
+   * without computing any subsets.
+   *
+   * @param a An element to compare with.
+   * @return Some element in this set equal to the one provided, or None.
+   */
+  public final Option<A> peek(final A a) {

```

Comment:
I think this function should be named either:
- `lookup`: to be consistent with `TreeMap#splitLookup` (and [haskell](https://hackage.haskell.org/package/containers-0.5.8.1/docs/Data-Set.html#v:lookupLT) ).
- or `get`: to be consistent with `TreeMap#get` (but which name is kind of inconsistent with `splitLookup`/haskell)

I would prefer `lookup`.